velox/exec/HashProbe.cpp

```diff
@@ -178,6 +182,43 @@ void HashProbe::initializeFilter(
   filterInputType_ = ROW(std::move(names), std::move(types));
 }
 
+void HashProbe::prepareForNullAwareAntiJoinWithFilter() {
```

Comment:
Storing a copy of the build side is going to double memory usage for the join. If there are no nulls on the probe side, we won't access non-null build side rows. Should we delay loading this data until we know we need it?